StantUiIosLibrary/Classes/Headers/SecondHeader/SecondHeaderButton.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SecondHeaderButton.swift
+//  StantUiIosLibrary
+//
+//  Created by Mac Mini Novo on 01/08/19.
+//
+
+import UIKit
+
+public struct SecondHeaderButton {
+    let action: Selector?
+    unowned var target: UIViewController
+    let icon: UIImage?
```

Comment:
This could also be a weak/unowned property

Reply:
Done.